packages/core/notifications/core-notifications-browser-internal/src/toasts/deduplicate_toasts.test.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { deduplicateToasts } from './deduplicate_toasts';
+import { Toast } from '@kbn/core-notifications-browser';
+
+const mockDismissToast = jest.fn();
+
+function toast(title: string, text: string, id = Math.random()): Toast {
+  return {
+    id: id.toString(),
+    title,
+    text,
+  };
+}
+
+describe('deduplicate toasts', () => {
```

Comment:
I think worth testing dismiss logic, react functions edge case, counter adding to the title here, wdyt?

Reply:
They're being tested in the test file for the whole list, but if we can fit it in this file, it would be better placed probably. I'll try to move those tests here.

Reply:
I wasn't sure if the other fail was testing all of this. If it was, then looks good.

Reply:
We could have a short sync call, and go through what cases are tested, what could be added